packages/katello/katello-repos/katello-repos.spec

```diff
@@ -5,7 +5,7 @@
 
 %global prereleasesource nightly
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 2
+%global release 1
 
 Name:           katello-repos
 Version:        4.10
```

Comment:
This can be bumped up to 4.11 since it's the new development version after branching

Reply:
You'll also need to update your comment.

The release version only goes down to 1 when we also bump the main version. Otherwise the release version should keep  going up.